tests/conftest.py

```diff
@@ -1,9 +1,10 @@
 import pytest
 
+from tests import tasks
 
-@pytest.fixture
-def default_worker_tasks(default_worker_tasks: set) -> set:
```

Comment:
By receiving the set from "itself", you can populate the set in a different fixture scopes to control which test has which tasks.

It's a feature not a bug :)